docs/builtins.md

```diff
@@ -1256,3 +1256,463 @@ x-axis and the ray from the origin to the point `x y`. More formally,
 it returns the arc tangent of `y/x` for given arguments `y` and `x`,
 using the signs of the two to determine the quadrant of the return
 value.
+
+## Graphics
+
+`evy` on the web outputs drawing commands to a drawing area
+called **canvas**.
+
+Positions on the canvas are defined by a coordinate system, similar to
+the Cartesian coordinate system used in mathematics. The horizontal
+dimension is called the **x-axis**, and the vertical dimension is
+called the **y-axis**.
+
+A point on the canvas is defined by its **x** and **y coordinates**,
+which are written as `x y`. For example, the point `30 60` has an
+x-coordinate of 30 and a y-coordinate of 60. It is located 30 units
+from the left edge of the canvas and 60 units from the bottom edge.
+
+<img width="300" alt="Catesian coordinates with x and y axes and point (30, 60)" src="img/coords.png">
+
+[Evy source](https://evy.dev/#content=H4sIAAAAAAAAE0WRwW6EIBCG7/MUE86NAV1w8dqkL7HxYJFtSVE2rmnXNn33DgL2NDMfA///h7fFjWCCDwuyVz+YDwZfblzfkVcnAO9mixwF5zCFz9jyxIhQC7fgN7zsQ48XfUZRydTEOfG8tL8SYSVRn/sIcokc4BrmFX/u7tt2J7wOk/Nbx17cMuBzGO0TTmEO99tgLPtNXrREAat9rMgeLCFBMKPtQGWJM0ioOVJQp3gJqY48hKUma7VG2VJtRK47z0t0qGKeus3LLSrRF56laomqzvq72OFBYK2L1e6fC5JQJVXXRJ4/Z7EjO/yTV+MW4+kC/AGSFdVbwgEAAA==).
+
+The canvas ranges from coordinates `0 0` to `100 100`. The center of the
+canvas has the coordinates `50 50`.
+
+### `move`
+
+`move` sets the position of the drawing pen to the given coordinates.
```

Comment:
The "drawing pen" (aka cursor) should be described in the Graphics preamble above. Also sometimes below you say "cursor position" and others "position of the drawing pen" and others "position of the pen". You should probably pick one and stick with it. "cursor" is more technical and probably doesn't make sense to new comers I guess.

Reply:
ok, done.
I kept cursor for reference and pen for descriptions and added an explanation to the intro.